sdk/core/azure-core/test/libcurl-stress-test/README.md

```diff
@@ -0,0 +1,29 @@
+# Stress test prototype 
```

Comment:
QQ: Is this a prototype or is it a template/example?

Prototype implies that it's an experiment, and this seems too complete to be a prototype.

Reply:
prototype of